1exercise4.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <title>Task</title>
+    
+    </head>
+<body>
+
+    <script>
+function countSum(salaries) {
+    sum = 0;
+    for (let key in salaries) {
+        
+        if (salaries[key] === undefined) {
+            sum;
+        } else {
+        sum += Number( salaries[key] );
+    }}
+    alert(sum);
```

Comment:
The function needs to return the sum. It's better to put the code responsible for displaying the sum outside the function.